src/include/ipc/topology.h

```diff
@@ -46,6 +46,7 @@ enum sof_comp_type {
 	SOF_COMP_DEMUX,
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
+	SOF_COMP_TDFB,		/**< Time domain fixed beamformer */
```

Comment:
This one will probably use the new process comp type, but @kv2019i will know for sure.

Reply:
UUID compatible now, resolved.